drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/navigation_notes_loader.rb

```diff
@@ -0,0 +1,79 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::TcHmis::Importers::Loaders
+  class NavigationNotesLoader < CustomAssessmentLoader
+    ASSESSMENT_DATE_COL = 'Date Taken'.freeze
+
+    CDED_CONFIGS = [
+      { element_id: 7313,
+        label: 'Are you assigned to a housing Navigator?',
+        key: 'nav_are_you_assigned_to_a_housing_navigator',
+        repeats: false,
+        field_type: 'string' },
+      { element_id: 7316, label: 'Case Notes', key: 'nav_case_notes', repeats: false, field_type: 'string' },
+      { element_id: 7356, label: 'Navigator.', key: 'nav_navigator', repeats: false, field_type: 'string' },
+      { element_id: 7357,
+        label: 'Did the client show up for appointment?',
+        key: 'nav_did_the_client_show_up_for_appointment',
+        repeats: false,
+        field_type: 'string' },
+      { element_id: 7358, label: 'Is this an attempt to contact?', key: 'nav_is_this_an_attempt_to_contact', repeats: false, field_type: 'string' },
+      { element_id: 7871, label: 'Time Spent', key: 'nav_time_spent', repeats: false, field_type: 'string' },
+      { element_id: 8487, label: 'Type of Client Interaction', key: 'nav_type_of_client_interaction', repeats: false, field_type: 'string' },
+      { element_id: 8841,
+        label: 'Problem Solving/Diversion/Rapid Resolution intervention or service result - Client housed/re-housed in a safe alternative',
+        key: 'nav_key_8841',
+        repeats: false,
+        field_type: 'string' },
+      { element_id: 8842, label: 'Referral Result', key: 'nav_referral_result', repeats: false, field_type: 'string' },
+      { element_id: 8843,
+        label: 'Referral to post-placement/follow-up case management result - Enrolled in Aftercare project',
+        key: 'nav_key_8843',
+        repeats: false,
+        field_type: 'string' },
+      { element_id: 8845, label: 'Date of Event', key: 'nav_date_of_event', repeats: false, field_type: 'string' },
+      { element_id: 8846, label: 'Date of result', key: 'nav_date_of_result', repeats: false, field_type: 'string' },
```

Comment:
These should match an existing CE Event that we have, lets discuss. We don't want to store the Event fields in CDEs. There are a few fields in addition to this one. Sorry I didn't flag earlier!

Reply:
per discussion in slack, CE event import is not needed